pkg/controller/jobs/job/job_controller.go

```diff
@@ -118,11 +117,11 @@ func (j *Job) Object() client.Object {
 }
 
 func (j *Job) ParentWorkloadName() string {
-	return j.Annotations[constants.ParentWorkloadAnnotation]
+	return j.Annotations[jobframework.ParentWorkloadAnnotation]
 }
 
 func (j *Job) QueueName() string {
-	return j.Annotations[constants.QueueAnnotation]
```

Comment:
I think this was lost over the rebases.... We should check first in Labels, and, if not found, as Annotation.

Reply:
Possibly, but if so, there wasn't a test for it, I'm going to add the support and the test, but I would like to move the function out from the interface as discussed here: https://github.com/kubernetes-sigs/kueue/pull/640#discussion_r1140250234.